src/main/java/com/google/codeu/servlets/AboutMeServlet.java

```diff
@@ -0,0 +1,67 @@
+package com.google.codeu.servlets;
+
+import java.io.IOException;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+import com.google.codeu.data.Datastore;
+
+/**
+ * Handles fetching and saving user data.
+ */
+// When user requests the /about URL, AboutMeServlet class is executed.
+@WebServlet("/about")
+public class AboutMeServlet extends HttpServlet {
+
+    private Datastore datastore;
+
+    @Override
+    public void init() {
+        datastore = new Datastore();
+    }
+
+    /**
+     * Responds with the "about me" section for a particular user.
+     */
+    @Override
+    public void doGet(HttpServletRequest request, HttpServletResponse response)
+            throws IOException {
+
+        response.setContentType("text/html");
+
+        String user = request.getParameter("user");
+
+        if(user == null || user.equals("")) {
```

Comment:
these keywords should be followed by an empty space for better readability

Reply:
I don't understand what you are asking, could you be a bit more specific please?